webextensions/api/browserAction.json

```
@@ -823,7 +823,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setPopup",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "4"
```

:x: https://github.com/chromium/chromium/commit/85ae959285a1ed1d423abd3d01cac5a71cd7b51f#diff-b71175023fc6d05ac325c35c2b3a915d0ca7c4b183e05ba49d11b451b34052f4 landed in [5.0.316.0](https://github.com/chromium/chromium/releases/tag/5.0.316.0) before Chrome 5.0.375.

```suggestion
                "version_added": "5"
```